exec/bigmachine.go

```diff
@@ -1091,7 +1104,10 @@ func (w *worker) CommitCombiner(ctx context.Context, key TaskName, _ *struct{})
 		case combinerCommitted:
 			return nil
 		case combinerError:
-			return errors.E("error while writing combiner")
+			if err := errors.Recover(w.combinerErrors[key]); err.Severity == errors.Fatal {
+				return maybeTaskFatalErr{errors.E("error while writing combiner", err)}
+			}
+			return errors.E("error while writing combiner", w.combinerErrors[key])
```

Comment:
I think we can just always wrap here...

Reply:
won't that make all errors fatal though since the reviseSeverity function doesn't check for maybeFatal and severity == fatal? LMK and I'll make the change accordingly.

Reply:
The coordinator node only aborts on fatal errors: https://github.com/grailbio/bigslice/blob/5593d488ac5e10b97d682902d133662da653a829/exec/bigmachine.go#L363-L366 .  Always wrapping here basically asserts that `w.combinerErrors[key]`'s severity applies to the task (as opposed to something else that should not be fatal to the task, e.g. fatal unavailability in bigmachine).